launch/README.md

```diff
@@ -0,0 +1,22 @@
+## Description
+Provides an easy way to set up a local chain based on .yml configs.
+For the Mangata nodes we are currently using the dev Docker image, which needs to be built locally.
+
+## Run
+build the mangata node & create image
```

Comment:
```suggestion
## Run
 Install `parachain-launch` globally

> yarn global add @open-web3/parachain-launch

Build the mangata node & create image
```

Reply:
we are not using the '@open-web3/parachain-launch' i had to fork and make some changes